docs/howto/deployment/wsgi/modwsgi.txt

```diff
@@ -97,34 +97,86 @@ should put in this file, and what else you can add to it.
 
 .. _daemon-mode:
 
-Using ``mod_wsgi`` daemon mode
-==============================
+Installing ``mod_wsgi`` using Python
+====================================
 
-"Daemon mode" is the recommended mode for running mod_wsgi (on non-Windows
-platforms). To create the required daemon process group and delegate the
-Django instance to run in it, you will need to add appropriate
-``WSGIDaemonProcess`` and ``WSGIProcessGroup`` directives. A further change
-required to the above configuration if you use daemon mode is that you can't
-use ``WSGIPythonPath``; instead you should use the ``python-path`` option to
-``WSGIDaemonProcess``, for example:
+To install ``mod_wsgi`` directly into your Python installation, from within the
+source directory of the ``mod_wsgi`` package you can run:
 
 .. code-block:: apache
 
-    WSGIDaemonProcess example.com python-home=/path/to/venv python-path=/path/to/mysite.com
-    WSGIProcessGroup example.com
+    python setup.py install
 
-If you want to serve your project in a subdirectory
-(``https://example.com/mysite`` in this example), you can add ``WSGIScriptAlias``
-to the configuration above:
+This will compile ``mod_wsgi`` and install the resulting package into your
+Python installation. If wishing to install an official release direct from
+the Python Package Index ``(PyPi)``, you can instead run:
 
 .. code-block:: apache
 
-    WSGIScriptAlias /mysite /path/to/mysite.com/mysite/wsgi.py process-group=example.com
+    pip install mod_wsgi
```

Comment:
Django style is to make this more verbose to help beginners avoid mistakes, see ticket-30367
```suggestion
    python -m pip install mod_wsgi
```

Reply:
Greetings. Still needs updating.